web/src/OverviewTable.tsx

```diff
@@ -532,6 +532,22 @@ const columnNameToInfoTooltip: {
       .
     </>
   ),
+  Widgets: (
```

Comment:
(maybe bikeshedding?) this tooltip text is making me wonder if this column should just be named `Custom actions`? i forgot what naming we originally talked about... but honestly, feel free to ignore this suggestion / thought.

Reply:
I'm not a fan of putting "custom" in the names of things, but haven't really had a good conversation about it and might be wrong. I'm happy to talk about it, but am also pretty sure it's not an urgent thing and we can change it later, so not going to block.